docs/source/serving/sky-serve.rst

```diff
@@ -521,3 +521,73 @@ The :code:`resources` field has the same spec as a normal SkyPilot job; see `her
   These settings will not take effect if you have an existing controller (either
   stopped or live).  For them to take effect, tear down the existing controller
   first, which requires all services to be terminated.
+
+Speedup Setup in Model Serving
+------------------------------
+
+When serving AI models, the setup process like dependencies installation and model weights downloading takes a lot of time on the cold-start time. According to one experiment we found that, dependencies installation can take 45% of times (271s out of 608s) of the setup process, and 16% for model weights downloading (98s out of 608s). To speed up this process, you can use the following methods:
+
+Use a Custom Image for VMs
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use a custom image that has the dependencies preinstalled (also probably model weights preloaded). This can be done by creating a VM, installing the dependencies, and then creating a custom image from the VM on the cloud console. Then, you can use the custom image to launch the VMs for serving:
```

Comment:
```suggestion
You can use a custom image that has the dependencies preinstalled and, optionally, model weights downloaded. This can be done by creating a VM, installing the dependencies, and then creating a custom image from the VM on the cloud console. Then, you can use the custom image to launch the VMs for serving:
```